extensions/positron-python/src/client/positron/session.ts

```diff
@@ -820,6 +827,36 @@ export class PythonRuntimeSession implements positron.LanguageRuntimeSession, vs
         }
     }
 
+    private enableAutoReloadIfEnabled(info: positron.LanguageRuntimeInfo): void {
+        // Enable auto-reload if the setting is enabled.
+        const configurationService = this.serviceContainer.get<IConfigurationService>(IConfigurationService);
+        const settings = configurationService.getSettings();
+        if (settings.enableAutoReload) {
+            // Enable module hot-reloading for the kernel.
+            const settingUri = `positron://settings/python.enableAutoReload`;
+            const banner = vscode.l10n.t(
+                'Automatic import reloading for Python is enabled. It can be disabled with \x1b]8;;{0}\x1b\\this setting\x1b]8;;\x1b\\.',
```

Comment:
```suggestion
                'Automatic import reloading for Python is enabled. It can be disabled with the \x1b]8;;{0}\x1b\\python.enableAutoReload setting\x1b]8;;\x1b\\.',
```

nit for clarity